package.json

```diff
@@ -2,7 +2,7 @@
   "name": "single-spa-react",
   "version": "5.1.4",
   "description": "Single-spa lifecycles helper for React apps",
-  "main": "lib/umd/single-spa-react.js",
+  "main": "lib/cjs/single-spa-react.cjs",
```

Comment:
Changing the `main` field from UMD to CJS is a breaking change. Please update the changeset to be a major

Reply:
Let's change this back - we can discuss the reasons for changing to CJS in Slack and possibly in another PR